### PR TITLE
Bh/1397 actor action filter

### PR DESCRIPTION
**Pull request by billyhunt**
[Link to ZenHub issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1397)

## What does this change?

As a user, it would be useful to be able to filter actions based on a given actor.  This change will add a side column in the actions section of the admin panel to filter actions based on actor. (defaults to all shown)

## Screenshots (for front-end PR):

image
image

## Checklist:

+ [x] Run ``` docker-compose run web python crt_portal/manage.py create_mock_reports 1000 ```
+ [x] Go to http://localhost:8000/admin/actstream/action/
+ [x] Click on the actors on the sidebar (USER_1, USER_2, USER_3, and whatever your super user is) and make sure the actions are filtering correctly with correct users, and the correct amount. 
+ [x] Try adding an action (change something on a report), and see if that action is added to the filter, and the number of actions for your super user is incremented.

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [ ] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/admin.py [hunk 1]
```diff
@@ -145,13 +146,39 @@ class ReportAdmin(ReadOnlyModelAdmin):
...
+
+    def queryset(self, request, queryset):
+        """
+        Returns the filtered queryset based on the value
+        provided in the query string and retrievable via
+        `self.value()`.
+        """
+        # Compare the requested value (either '80s' or '90s')
+        # to decide how to filter the queryset.
+        if not self.value():
```
Comment (JackRyan1989) [inline crt_portal/cts_forms/admin.py:169]: So we're just checking if self.value is truthy here right? 

This should boil down to: if there is no provided value to the ActorActionFilter, then we just return the queryset, if there is a provided value then we filter the queryset by the provided value, correct?

**Review by JackRyan1989 — APPROVED** [review]
Tested on FE, works a ok for me. Good job, Billy!

File: crt_portal/cts_forms/admin.py (hunk 1 above)
Comment (billyhunt) [reply crt_portal/cts_forms/admin.py:169]: Exactly!

Comment (billyhunt) [reply crt_portal/cts_forms/admin.py:169]: Otherwise it does the filter.

**Pull request merged**